content/brazilian-portuguese/java-basics/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Java: Fundamentos"
+description: "Uma introdução aos fundamentos da linguagem de programação Java"
+date: 2021-10-13T00:00:00Z
+prereq: "Nenhum"
+difficulty: "Iniciante"
+icon: "fab fa-java"
+draft: false
+---
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/udEaocEYlpc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+Bem vindo ao curso <b>Java: Fundamentos</b>, onde você vai aprender uma linguagem de programação amplamente utilizada e de alto nível, que fornece ao computador instruções para criar programas emocionantes!
+
+Java é uma linguagem de programação de alto nível desenvolvida por James Gosling e Sun Microsystems na Califórnia em 1995. 
+
+<img src="images/logo.png" height="150" alt="Java Logo of steaming coffee cup with Java written below" />
```

Comment:
Can we translate all the image alt texts as well?

i.e. translate Java Logo of steaming coffee cup with Java written below

Reply:
I did not suggest it because I did not see where it was shown on the page. Do you still think we need to translate it?